Library/Homebrew/dev-cmd/update-test.rb

```diff
@@ -148,4 +140,17 @@ def update_test
   ensure
     FileUtils.rm_rf "update-test" unless args.keep_tmp?
   end
+
+  def git_tag
```

Comment:
```suggestion
  def git_tags
```

because it returns multiple